Python/Product/Cookiecutter/Model/GitClient.cs

```diff
@@ -83,25 +89,31 @@ class GitClient : IGitClient {
 
                 return localTemplateFolder;
             }
-        }   
-        
-        public bool ExistsOnPath(string fileName) {
-            return GetFullPath(fileName) != null;
         }
 
-        public string GetFullPath(string fileName) {
-            if (File.Exists(fileName)) {
-                return Path.GetFullPath(fileName);
-            }
-            var values = Environment.GetEnvironmentVariable("PATH");
-            foreach (var path in values.Split(Path.PathSeparator)) {
-                var fullPath = Path.Combine(path, fileName);
-                if (File.Exists(fullPath))
-                    return fullPath;
+        public bool canRunGit(string exe) {
+            var process = new Process {
```

Comment:
`Process` is disposable, so this needs a `using` variable declaration or block to ensure cleanup.